ee/experimental/graph_utils.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+import networkx as nx
+
+from keep.api.core.db import get_pmi_value
+
+
+def detect_knee_1d_auto_increasing(y):
+    """
+    Detect the knee point in an increasing 1D curve automatically.
```

Comment:
As someone who has no idea what's the knee point, perhaps add some docs on what does that mean? what are we looking to understand here?